### ISSUE TITLE
The code is hard to read and make it hard to follow . where can I get an entrance of the code

### ISSUE THREAD

**Issue by db12138** [harsh19/Reasoning-Chains-MultihopQA#1]
the program start bu python -m allennlp.run . It's hard to debug on server

**Comment by harsh19** [comment]
Hi, Sorry you feel that way. The code does have dependency on allennlp, and the model training uses commands such as allennlp.run train (http://docs.allennlp.org/v0.9.0/api/allennlp.commands.html). However, the model and evaluation codes are available in code/allennlp_reasoning_explainqa/ of this repository.  So, could you please clarify more on the exact issue you are facing -  did you face any runtime error ? What were you trying to debug ?
